t/ui/10-tests_overview.t

```diff
@@ -497,8 +498,8 @@ subtest 'filtering by job group' => sub {
             name => 'SLE 15 SP5 development'
         });
 
-    my $get_text = sub {
-        $driver->get(shift);
+    my $get_text = sub ($url) {
```

Comment:
Isn't this just a style change? I don't see anything here that changes behavior

Reply:
Yes, this is just a style change. I first thought I could reuse this function so I changed it to our current style as I had to move it around. As it turned out that I couldn't reuse it I still kept the style change (as separate commit).

Reply:
I suggest renaming the PR then

Reply:
Hrm. Looking at the commits `return filter.replace(/_/g, ' ');` is a functional fix, though?

Reply:
Okay, so I guess the part I was missing is this the `return` is required if you also use curly parens

Reply:
Yes. The functional and the style changes are in separate commits so this should be clear.

Reply:
Well, it wasn't 😅 I mentioned it because it stopped me approving, and sometimes you ask about unreviewed PR's